pkg/commands/update/versioner.go

```diff
@@ -101,91 +81,83 @@ func (g GitHub) LatestVersion(ctx context.Context) (semver.Version, error) {
 func (g GitHub) Download(ctx context.Context, version semver.Version) (filename string, err error) {
```

Comment:
Yeah, the named return params are just creating ambiguity here, as evidenced by all the 

```go
return filename, <non-nil error>
```

blocks in the method.  Should just remove them e.g. `(string, error)`.

Reply:
Thanks, will do.